README.md

```diff
@@ -59,10 +59,10 @@ npm i es-check --save-dev
 Check if an array or glob of files matches a specified ES version
 
 ```sh
-node_modules/es-check/index.js check es5 ./vendor/js/*.js ./dist/**/*.js
+es-check check es5 ./vendor/js/*.js ./dist/**/*.js
```

Comment:
lol this seems redundant to me

Reply:
actually, i am going to remove the `check` subcommand to make this an actual breaking change